Tests/SwiftDocCTests/Indexing/RenderIndexTests.swift

```diff
@@ -96,7 +96,7 @@ final class RenderIndexTests: XCTestCase {
                             "type": "groupMarker"
                           },
                           {
-                            "path": "/documentation/mixedlanguageframework/bar/mystringfunction(_:)",
+                            "path": "/documentation/mixedlanguageframework/bar/mystringfunction(__)",
```

Comment:
@ethan-kusters do you remember if this value represents the path of the web URL or the file path within the documentation archive? I _thought_ that this is the path that the browser would load to access this page.